c/meson.build

```diff
@@ -18,6 +18,10 @@ nns_capi_single_srcs += join_paths(meson.current_source_dir(), 'src', 'ml-api-in
 nns_capi_pipeline_srcs += join_paths(meson.current_source_dir(), 'src', 'ml-api-inference-pipeline.c')
 nns_capi_service_srcs += join_paths(meson.current_source_dir(), 'src', 'ml-api-service-db.cc')
 
+if get_option('enable-machine-learning-agent')
+  nns_capi_service_srcs += join_paths(meson.current_source_dir(), 'src', 'ml-api-service-agent.cc')
+endif
```

Comment:
nns_capi_service_srcs are the ml-service shared library.
Don't you need a meson entry for an executable (daemon) with ml-api-service-agent.cc?